src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUsers } from '../../services/getUsers';
+
+type Props = {
+  todo: Todo
+  setSelectedTodo: (selectedTodo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, setSelectedTodo }) => {
+  const [loaded, setLoaded] = useState(false);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    setLoaded(true);
+    getUsers(todo.userId)
+      .then(u => {
```

Comment:
Don't cut variable names, try to rename to more meaningful variable name
```suggestion
      .then(userItem => {
```